src/mixins/amalgamation-mixin.ts

```diff
@@ -119,7 +119,7 @@ export default class AmalgamationMixin extends Vue {
   xproUlcCcc (business: AmalgamatingBusinessIF): AmlStatuses {
     if (
       business.type === AmlTypes.FOREIGN &&
-      (!this.isTypeBcUlcCompany || !this.isTypeBcCcc)
+      (this.isTypeBcUlcCompany || this.isTypeBcCcc)
```

Comment:
I believe the logic here is incorrect. I've been playing around with this and noticed it's incorrect.

We want to show this rule when the type is ULC or CCC not the other way around. I've been getting this rule when I was on a Limited company amalgamation with Foreign companies.

Please correct me if I'm wrong.

Reply:
I think you are correct (as per the comment on line 118) 👍 

Thanks for playing around and verifying stuff :)